File: zoho_preparation_a1b10.java
// Eg 1: Input: a1b10
//        Output: abbbbbbbbbb
// Eg: 2: Input: b3c6d15
//           Output: bbbccccccddddddddddddddd
// The number varies from 1 to 99.

import java.util.*;
class test{
	public static void main(String[] args) {
		
		String input = "b3c6d15";
		char inps[] = input.toCharArray();
		String con = "";
		int i=0;

		for(char a : inps)
		{
			
			if(Character.isAlphabetic(a))
			{
				for(int j= i+1; j <= i+2;j++)
				{
					if(Character.isAlphabetic(inps[j]))
					{
						break;
					}
					else
					{
						con = con + inps[j];
					}
				}

				//Printing

				for(int k = 0; k < Integer.valueOf(con); k++)
				{
					System.out.print(a);
				}
			}


			i++;
			con = "";
		}	


	}
}

File: zoho_preparation_a1b10.py
# // Eg 1: Input: a1b10
# //        Output: abbbbbbbbbb
# // Eg: 2: Input: b3c6d15
# //           Output: bbbccccccddddddddddddddd
# // The number varies from 1 to 99.

inps = "b3c6d15"
ite = 0
val = ""

for i in inps:
    if i.isalpha():
        for j in range(ite+1 ,ite+3):
            if inps[j].isalpha():
                break;
            else:
                val = val + str(inps[j])


        for k in range(0, int(val)):
            print(i, end="")
    
    ite = ite + 1
    val= ""